includes/emails/email-summary/edd-email-summary-template.php

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+		<html>
```

Comment:
It will affect the entire file, but not a big deal--but can we un-indent this one level?